advanced/Scripts/piholeDebug.sh

```diff
@@ -151,6 +160,17 @@ function hostnameCheck {
     fi
 }
 
+function portCheck {
+    echo "############################################################" >> ${DEBUG_LOG}
+	echo "########           Open Port Information           #########" >> ${DEBUG_LOG}
+	echo "############################################################" >> ${DEBUG_LOG}
+
+    echo "::: Writing local server ports to logfile"
+
+    ${SUDO} netstat -tulpn >> ${DEBUG_LOG}
```

Comment:
does `netstat` require installing netstat on any of the officially supported OSs?

I know you don't support alpine but it lacked netstat by default in my docker container and I didn't want to add time to my tests so decline installing it and instead just `curl`'d 80 and ran `dig` for port 53

Dig test: https://github.com/diginc/docker-pi-hole/blob/master/test/test_start.py#L36
Curl test: https://github.com/diginc/docker-pi-hole/blob/master/test/test_start.py#L46